tests/baselines/reference/tslibFindAllReferencesOnRuntimeImportWithPaths1.baseline.jsonc

```diff
@@ -0,0 +1,38 @@
+// === findAllReferences ===
+// === /tests/cases/fourslash/project/src/foo.ts ===
+// [|<|import * as x from /*FIND ALL REFS*/"[|tslib|]";|]|>
```

Comment:
Since we are already highlighting import do we really want first statement as well ?

Reply:
Yep - there's two references in this file, after all - the implicit one and the explicit one.